docs/timelines/timeline-options.md

```diff
@@ -0,0 +1,104 @@
+---
+id: timeline-options
+title: Timeline Options
+sidebar_label: Timeline options
+---
+
+Timelines have a number of options which set how they operate. 
+Press \<Options\> (or the \[Options\] softkey on the top level menu) then the **select** button for the timeline you wish to edit.
+
+--- 
+
+## Handle Tab
+
+All options are the same as for a cue, see [Options](../cues/playback-options.md#handle-tab).
+
+--- 
+
+## Times Tab
+
+![Timeline Options: Times](/docs/images/Timeline-Options-Times.png)
+
+These options let you set the duration and start time of the timeline. These set the loop points when you are using the 
+[Loop function](../timelines/timeline-options.md#timecode-tab) or are used to fire the timeline automatically when the timecode is in the correct range.
+They can also be used to set limits on the timeline so you can be sure nothing will happen if you get sent some wayward timecode.
+
+--- 
+
+## Fader Tab
+
+![Timeline Options: Fader](/docs/images/Timeline-Options-Fader.png)
+
+Sets what happens to an internal timecode source when the playback fader containing the timeline is raised or zeroed. If an external
+timecode source is being used then this option has no effect.
+
+Option              | Action
+--------------------|---
+Fader Raised Action | **No Action**: The internal timecode source is not changed (default).<br>Play: The internal timecode source is started.
+Fader Zero Action   | **No Action**: The internal timecode source is not changed (default).<br>Stop: The internal timecode source is stopped. <br>Pause: The internal timecode source is paused at the current time.
+
+> This option is useful for creating a complicated one-shot effect for busking. Create the effect as a timeline using internal timecode and set these options to **Play**/**Stop** - you can fire your effect just by raising the fader and when you lower the fader to zero the effect will reset.
+
+--- 
+
+## Release Tab
```

Comment:
Maybe worth clarifying as per other comment on this that `Timeline-Global Release` replaces your user's `Global Release` settings when things are done by the timeline.